src/common/nodes/properties/Property.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * base interface of a property, this can be a propety on the many or on the one side
+ */
+export interface Property<T> {
```

Comment:
What the type?!

Reply:
added documentation with 9fbc9002590d6517d635bed793ac2e6f007f933e
the type is not constrained because a property can contain elements of any type